src/Gui/InputField.cpp

```diff
@@ -175,12 +186,10 @@ void InputField::updateText(const Base::Quantity& quant)
     setText(QString::fromStdString(txt));
 }
 
-void InputField::resizeEvent(QResizeEvent *)
+void InputField::resizeEvent(QResizeEvent * /*event*/)
 {
-    QSize sz = iconLabel->sizeHint();
-    int frameWidth = style()->pixelMetric(QStyle::PM_DefaultFrameWidth);
-    iconLabel->move(rect().right() - frameWidth - sz.width(),
-                    rect().center().y() - sz.height() / 2);
+    QSize iconSize = iconLabel->sizeHint();
+    iconLabel->move(width() - (iconSize.width() + 2 * getMargin()), (height() - iconSize.height()) / 2);

```

Comment:
I keep wondering why this `2 * getMargin()`, but if you tested the code, then sure!